test/features/publishers_home_test.rb

```diff
@@ -65,13 +65,37 @@ class PublishersHomeTest < Capybara::Rails::TestCase
     refute_content page, 'Pending: Email address has been updated'
   end
 
+  test "publisher with unverified channel has a dialog popup with a close" do
+    publisher = publishers(:small_media_group)
+    channel = channels(:small_media_group_to_verify)
+
+    sign_in publisher
+    visit home_publishers_path
+    assert_content page, "Verification Failed"
+
+    # Close out of partial
+    find("#modal-close-button").click
+    assert_no_content page, "Verification Failed"
```

Comment:
it would also be good to confirm the modal is dismissed permanently, i.e. `#disable_verification_failed_modal` runs correctly and `#shown_verification_failed_modal` is updated.